src/analytics/loadingPage/useLogDownloadAnalytics.ts

```diff
@@ -13,11 +14,11 @@ interface P extends Properties {}
 interface Analytics extends A<Action> {}
 
 export const useLogDownloadAnalytics = (): Analytics => {
```

Comment:
Looks great! I think we can use it here too
```suggestion
export const useLogDownloadAnalytics = () =>
   useAnalyticsRoot<Action>("LoadingPage");
```